src/App.js

```diff
@@ -53,17 +55,87 @@ const App = () => {
     setResults(workout);
   };
 
+  const [removeExercises, setEx] = useState(null);
+  const removeEx = (workout) => {
```

Comment:
I would go the full way and use the name removeExercise so it's even more obvious what is going on here (for a new person looking at your code, they would have to, even for a moment think what it is for.

Reply:
Got it!